src/content/docs/logs/forward-logs/kubernetes-plugin-log-forwarding.mdx

```diff
@@ -43,76 +57,104 @@ To forward your Kubernetes logs to New Relic with our plugin:
 
 New Relic has a [Fluent Bit](https://fluentbit.io/) [output plugin](https://github.com/newrelic/newrelic-fluent-bit-output) to forward your logs to New Relic log management. This plugin is also provided in a standalone Docker image that can be installed in a Kubernetes cluster in the form of a DaemonSet, also known as the Kubernetes plugin.
 
-You can install it in your cluster using our Helm chart in two ways.
+You can install it in your cluster using our Helm chart in two ways:
 
-### Use our guided install [#helm-guided-install]
-Although the [newrelic-logging](https://github.com/newrelic/helm-charts/tree/master/charts/newrelic-logging) chart works as a standalone, we recommend installing it as part of the [nri-bundle](https://github.com/newrelic/helm-charts/tree/master/charts/nri-bundle) chart. 
 
-The best way to install this is through our [guided installation](/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure/) process. This guided install can generate the Helm 3 commands required to install it ([see "Helm 3"](/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure/#finish-your-install)). 
+<CollapserGroup>
+    <Collapser
+        id="helm-guided-install"
+        title="Use our guided install"
+    >
+    Although the [`newrelic-logging`](https://github.com/newrelic/helm-charts/tree/master/charts/newrelic-logging) chart works as a standalone, we recommend installing it as part of the [`nri-bundle`](https://github.com/newrelic/helm-charts/tree/master/charts/nri-bundle) chart. 
 
-### Manual installation [#manual-helm-install]
+    The best way to install this is through our [guided install](/install/kubernetes/?dropdown1=guided) process. This guided install generates the command required to install it. 
 
-Alternately, you can install it manually using Helm, by running this command to install the repo:
+    </Collapser>
+    <Collapser
+        id="manual-helm-install"
+        title="Manual installation using Helm"
+    >
 
-```bash
-    helm repo add newrelic https://helm-charts.newrelic.com
-```
-To update the repo you can run:
+    Alternately, you can install it manually using Helm. Run this command to install the repo:
 
-```bash
-    helm repo update newrelic
-```
+    ```bash
+        helm repo add newrelic https://helm-charts.newrelic.com
+    ```
+    To update the repo you can run:
+
+    ```bash
+        helm repo update newrelic
+    ```
 
-Go here for [uninstallation instructions](/docs/kubernetes-pixie/kubernetes-integration/uninstall-kubernetes/).
+    </Collapser>
+</CollapserGroup>
+
+
+Go here for [uninstallation instructions](/docs/kubernetes-pixie/kubernetes-integration/installation/uninstall-kubernetes/).
 
 ## Troubleshoot your Kubernetes plugin for log forwarding installation [#troubleshoot-installation]
```

Comment:
@nbaenam this section seems to cover both the Kubernetes log forwarding plugin and the Fluent Bit approach. If that's the case, do you need to broaden this heading to include both?

Reply:
I'll check it out with Mikel.